ReactWindows/ReactNative/Views/Text/ReactTextViewManager.cs

```diff
@@ -24,6 +26,27 @@ public override string Name
         }
 
         /// <summary>
+        /// Sets the text alignment.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="textAlign">The text alignment string.</param>
+        [ReactProp(ViewProps.TextAlign)]
+        public void SetTextAlign(TextBlock view, string textAlign)
```

Comment:
Something tells me this may have to go on the shadow node.  I know it probably shouldn't but it seems that it could affect the measurements.